data/ledger_test.go

```diff
@@ -125,7 +125,7 @@ func TestLedgerCirculation(t *testing.T) {
 	cfg := config.GetDefaultLocal()
 	cfg.Archival = true
 	log := logging.TestingLog(t)
-	log.SetLevel(logging.Warn)
+	log.SetLevel(logging.Panic)
```

Comment:
Why is this being upgraded to panic? I see it is only used to be passed to the `realLedger`, so I guess if realLedger has anything to say, this test panics, is that right?

Reply:
This will print many many lines with tested with -v flag. Changing it to `Panic` does the trick.

Reply:
This is not a solution, please revert. The fix is https://github.com/algorand/go-algorand/pull/4856/files#diff-cfa2b2e90d28d3b0d2a4c658377ed1be5e4845a0a430799a0938eaae8c33d433R26 just merged into master